src/extension/sdk/dev_tools/manager.ts

```diff
@@ -98,22 +98,18 @@ export class DevToolsManager implements vs.Disposable {
 
 		if (!this.devtoolsUrl) {
 			this.devToolsStatusBarItem.hide();
-			// Don't try to check for install/version if we're using a custom script.
-			if (!this.workspaceContext.config?.devtoolsActivateScript) {
```

Comment:
I presume the removal of this means it's now ok to use `pub global activate devtools` and `pub global run devtools` for these workspaces?

Reply:
We can, although optimally for internal applications we can use `daemonScript` to launch a flutter daemon and then request DevTools to start from there. There are some changes planned for DevTools launch more generally, so I don't think this is a priority and perhaps we should continue using `pub global` for now.